src/did/src/mint_order_exemption.rs

```diff
@@ -0,0 +1,172 @@
+use candid::{CandidType, Principal};
+use ethers_core::abi::ethabi::Bytes;
+use ethers_core::abi::{Error, Function, Param, ParamType, StateMutability, Token};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+
+use crate::H256;
+
+/// Function signature, used if transaction should notify
```

Comment:
Isn't it pretty much the same as `NOTIFICATION` (expect of the name). Do we need to have a different one?

Reply:
Yeah, you're right. Should we keep just `MintOrderExemptionUserData` then? @GraDKh